### ISSUE TITLE
Firestore emulator: committing empty batches

### ISSUE THREAD

**Issue by merlinnot** [firebase/quickstart-nodejs#63]
I was instructed by @samtstern to post Firestore Emulator - related issues in this repository, so here we go:

Committing empty batches results in an error message
```bash
2 UNKNOWN:
```

Emulator output:
```
API endpoint: http://0.0.0.0:8080
If you are using a library that supports the FIRESTORE_EMULATOR_HOST environment variable, run:

   export FIRESTORE_EMULATOR_HOST=0.0.0.0:8080

Dev App Server is now running.

Mar 11, 2019 6:14:01 PM io.gapi.emulators.grpc.GrpcServer$3 operationComplete
INFO: Adding handler(s) to newly registered Channel.
Mar 11, 2019 6:14:01 PM io.gapi.emulators.netty.HttpVersionRoutingHandler channelRead
INFO: Detected HTTP/2 connection.
Mar 11, 2019 6:14:02 PM com.google.cloud.datastore.emulator.impl.util.WrappedStreamObserver onError
INFO: operation failed: No value present
```

Repro:
```ts
import { Firestore } from '@google-cloud/firestore';
import { credentials } from 'grpc';

const firestore = new Firestore({
  'grpc.initial_reconnect_backoff_ms': 500,
  'grpc.max_reconnect_backoff_ms': 1000,
  port: 8080,
  projectId: 'test',
  servicePath: 'localhost',
  sslCreds: credentials.createInsecure(),
});

const run = async () => {
  await firestore.batch().commit();
};

run().catch(x => console.error(x.message));
```

**Comment by samtstern** [comment]
Thanks @merlinnot !

**Comment by ryanpbrewster** [comment]
Fix is out for review, I'll update this bug when it gets released.

**Comment by merlinnot** [comment]
The issue is resolved in v1.4.1, thank you.
